src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -35,9 +35,7 @@ public LiftSubsystem(final RobotMap.LiftMap map) {
         brake = map.getBrake();
         armMotor = map.getArmMotor();
         heightEncoder = map.getHeightEncoder();
-        lowerLimit = map.getLowerLimit();
-        upperLimit = map.getUpperLimit();
-        manipAngle = map.getManipAngle();
+
```

Comment:
Why were `lowerLimit` and `upperLimit` removed, if they're used later on in the file?

Reply:
It will build, but crash when running because it's accessing a null pointer. The limit switches are part of lift, not maflipulator, and should not be removed